libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/crop/CropFragment.kt

```diff
@@ -57,11 +59,15 @@ class CropFragment : Fragment(), UCropFragmentCallback {
     private fun initializeViewModels() {
         viewModel = ViewModelProvider(this).get(CropViewModel::class.java)
         setupObservers()
-        viewModel.start(navArgs.inputFilePath, navArgs.outputFileExtension, requireContext().cacheDir)
+        viewModel.start(
+            navArgs.inputFilePath,
+            navArgs.outputFileExtension,
+            File(requireContext().cacheDir, "media_editing")
```

Comment:
We might consider passing just the `requireContext().cacheDir` as it requires Context and creating the subfolder within the VM, wdyt? We might also consider creating a constant for the `media_editing`. Wdyt?

Reply:
Done here 5d57f28